### PR TITLE
Create dataOracles.md

### PR DESCRIPTION
**Pull request by mirk26**
Adding WYND Environmental data and oracles services lite paper

### CODE REVIEW

File: oracles/dataOracles.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+cases, and are sometimes not easily comparable with other metrics and
+algorithms. Many scientific data products do not follow common data
+protocols, or even have a clean API to query for another task. Finally,
+and perhaps most importantly, the collaborative sharing of data and
+algorithms often stalls due to the challenge of incentivizing and
+rewarding continued interest and community development. The
+inconsistency of funding often precipitates the breakdown of
+relationships between data users and providers due to funding
+constraints, which eventually renders data sets and data platforms
+obsolete to operational management decisions and environmental action.
```
Comment (ethanfrey) [inline oracles/dataOracles.md:28]: Great paragraph.

This last sentence, "... data platforms obsolete **due** to operational management..." ? 
Not sure if that is the right fix, but something needs changing there

File: oracles/dataOracles.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+5)  easy collaboration
+
+This approach provides a foundation that will enable a multitude of
+conservation and business applications focused on ecological health, as
+well as turbo-charging research and analysis to advance the state of the
+art.
+
+**Open Data**
+
+**One essential aspect of this project is that all source data is
```
Comment (ethanfrey) [inline oracles/dataOracles.md:49]: Great call to action

File: oracles/dataOracles.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
+**[WYND Environmental Data & Oracles Service]{.ul}**
```
Comment (ethanfrey) [inline oracles/dataOracles.md:1]: Can you make this a header `#` not `**`

Also, there is metadata to add. Maybe you just put all this in index.md and use the header from that

File: oracles/dataOracles.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+4)  free exchange
+
+5)  easy collaboration
+
+This approach provides a foundation that will enable a multitude of
+conservation and business applications focused on ecological health, as
+well as turbo-charging research and analysis to advance the state of the
+art.
+
+**Open Data**
```
Comment (ethanfrey) [inline oracles/dataOracles.md:47]: Please use `##` as subheader

File: oracles/dataOracles.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+However, many projects uploading "ground truth" need continuous funding
+to be sustainable. While we require all remote sensing data to be under
+an open licence, we will allow such user-uploaded data to be non-open
+source, as many groups are concerned about reuse without payments.
+
+To provide a viable revenue stream for all data providers, we focus on
+value-added products on top of the data. Other teams can build products
+directly on the basic data for free, or pay to use these extensions.
+
+> **Data Value Enrichment**
```
Comment (ethanfrey) [inline oracles/dataOracles.md:72]: Please use `##` as subheader

File: oracles/dataOracles.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+WYND will collect user specific data and other open data sources and
+perform the process of Extracting, Transforming and Loading data on base
```
Comment (ethanfrey) [inline oracles/dataOracles.md:75]: `... Loading data on **a** base map..."

... (H3 Index) of **the** base map"

File: oracles/dataOracles.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+represent a shareable user and site behaviour which changes over time.
+This brings unprecedented visibility to the whole community about the
+site and the data publishers.
+
+Together the WYND aims to present a framework that serves as a market
+for innovation, and which will evolve as observing techniques and
+algorithms advance. Still, innovation will be contained within 3 basic
+categories for descriptors and metrics of ecosystem health:
+
+1)  Carbon / Living Biomass: these metrics describe the total mass of
```
Comment (ethanfrey) [inline oracles/dataOracles.md:90]: I think you need `1. ` to make a proper "ordered list" in markdown.

File: oracles/dataOracles.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+these metrics may change in isolation of the others. For instance, while
+it is likely that many of these metrics would be consistent or
+correlated for an impacted region (e.g. the city of Los Angeles), it is
+possible that environmental action may shift only one (e.g. with a
+conversion to electric vehicles, the pollution metric alone may
+improve). As we consider a rapidly changing ecosystem, it is likely that
+these metrics could evolve independently or in sequence but may not be
+temporally or spatially correlated.
+
+---\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\--
```
Comment (ethanfrey) [inline oracles/dataOracles.md:121]: Everything above this line is 💯 🎖️ 
Please put that (with the spellchecking feedback) into `index.md`

Let's pull the text below this section into it's own page called "use cases" or such.

The  part above here is a great summary and convincing pitch

**Review by ethanfrey — APPROVED** [review]
Good stuff. Content is impressive.

Left some comments on syntax and organisation mainly.
Please address them and merge

**Comment by mirk26** [comment]
Resolved comments from Ethan, mostly markdown, removed use case scenario.

**Pull request merged**